.github/workflows/inspektor-gadget.yml

```
@@ -78,100 +190,303 @@ jobs:
+    name: Build local gadget for all platforms
+    # level: 1
+    needs: checkout-code
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # TODO add local-gadget-linux-arm64
+        local-gadget-target: [local-gadget-linux-amd64]
+    steps:
+    - name: Setup go 1.17
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.17
+      id: go
+    - name: Cache go 1.17
+      uses: actions/cache@v2
+      with:
+        path: |
+          ~/.cache/go-build
+          ~/go/pkg/mod
+        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-go-
+    - name: Cache debian packages
+      uses: actions/cache@v2
+      id: cache-debs
+      with:
+        path: "~/cache-debs"
+        key: cache-debs-libbpf-1:0.4.0-1ubuntu1-libseccomp-2.5.1-1ubuntu1~20.04.1
+    - name: Install debian packages
+      env:
+        CACHE_HIT: ${{steps.cache-debs.outputs.cache-hit}}
       run: |
-        make manifests generate generate-documentation
-        git diff --exit-code HEAD --
+          if [[ "$CACHE_HIT" == 'true' ]]; then
+            sudo cp --verbose --force --recursive ~/cache-debs/* /
+          else
+            sudo apt install -y software-properties-common
+            sudo add-apt-repository -y ppa:tuxinvader/kernel-build-tools
+            sudo apt-get update
+            sudo apt install -y libbpf-dev="$LIBBPF_VERSION" libseccomp-dev="$LIBSECCOMP_VERSION"
+            mkdir -p ~/cache-debs
+            sudo dpkg -L libbpf-dev libseccomp-dev | \
+                while IFS= read -r f; do \
+                    if test -f $f; then echo $f; fi; \
+                done | xargs cp --parents --target-directory ~/cache-debs/
+          fi
+    - name: Get inspektor-gadget from artifact.
+      uses: actions/download-artifact@v2
+      with:
+        name: inspektor-gadget-tar
+        path: /home/runner/work/inspektor-gadget/
+    - name: Build ${{ matrix.local-gadget-target }}
+      run: |
+        tar -xf /home/runner/work/inspektor-gadget/inspektor-gadget.tar
+        cd inspektor-gadget
 
+        make ${{ matrix.local-gadget-target }}
+
+        # Prepare assets for release and actions artifacts
+        platform=$(echo ${{ matrix.local-gadget-target }} | cut -d- -f3-4)
+        mkdir $platform
+        cp ${{ matrix.local-gadget-target }} $platform/local-gadget
+        cp LICENSE $platform/
+        tar --sort=name --owner=root:0 --group=root:0 \
+          -czf ${{ matrix.local-gadget-target }}.tar.gz -C $platform \
+          local-gadget LICENSE
+        rm -rf $platform
+    - name: Add ${{ matrix.local-gadget-target }}.tar.gz as artifact.
+      uses: actions/upload-artifact@master
+      with:
+        name: ${{ matrix.local-gadget-target }}-tar-gz
+        path: /home/runner/work/inspektor-gadget/inspektor-gadget/inspektor-gadget/${{ matrix.local-gadget-target }}.tar.gz
+
+  build-docker-image:
+    name: Build inspektor-gadget docker image
```

```suggestion
    name: Build gadget container image
```

(it is not only docker anymore)